lib/nerves_init_gadget/options.ex

```diff
@@ -5,5 +5,6 @@ defmodule Nerves.InitGadget.Options do
             address_method: :linklocal,
             mdns_domain: "nerves.local",
             node_name: nil,
-            node_host: :mdns_domain
+            node_host: :mdns_domain,
+            ssh_console_port: 22
```

Comment:
Did you want to enable ssh by default?

Reply:
Whoops! nope i don't think that is a good idea..